test/ambuda/conftest.py

```diff
@@ -116,11 +128,15 @@ def initialize_test_db():
     session.add(admin_role)
     session.flush()
 
+    u_p1.roles = [p1_role]
```

Comment:
What are `p1_role` and `p2_role`? We have a nice descriptive name for `admin_role`, but the other two are not self-explanatory. We should consider to change them.

Reply:
BTW, above comment should not hold up this PR.